rustls/src/msgs/handshake_test.rs

```diff
@@ -811,6 +819,14 @@ fn get_sample_certificatepayloadtls13() -> CertificatePayloadTls13<'static> {
     }
 }
 
+fn get_sample_compressedcertificate() -> CompressedCertificatePayload<'static> {
```

Comment:
ordering nit: I think this should be below `get_sample_serverkeyexchangepayload_ecdhe()`.

Reply:
> i think the current ordering is in keeping with the rest of the file (which is extremely bottom-up).

Reasonable :+1: